axum-extra/Cargo.toml

```diff
@@ -67,7 +68,7 @@ prost = { version = "0.13", optional = true }
 serde_html_form = { version = "0.2.0", optional = true }
 serde_json = { version = "1.0.71", optional = true }
 serde_path_to_error = { version = "0.1.8", optional = true }
-tokio = { version = "1.19", optional = true }
+tokio = { version = "1.19", optional = true, features = ["fs"] }
```

Comment:
Shouldn't this feature only be enabled when `file-stream` is enabled?

Reply:
> Shouldn't this feature only be enabled when `file-stream` is enabled?

Yes, I fixed it.